src/pages/Login.js

```diff
@@ -0,0 +1,103 @@
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+
+export default function Login() {
+  const [user, setUser] = useState([]);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [msg, setMsg] = useState("");
```

Comment:
Instead of having multiple states you can use a single state having a object that will help you ease out while setting the values related to the values present in the keys of that object!!